src/features/events/components/EventsList/index.tsx

```diff
@@ -21,15 +27,15 @@ export const EventsList: FC = () => {
 
   const events = useMemo(() => {
     if (filter === FilterType.ALL) {
-      return data.sort((a, b) => (a.startsAt < b.startsAt ? -1 : 1))
+      return [...data].sort(sorts.ascending)
     } else if (filter === FilterType.FUTURE) {
       return data
         .filter((event) => isAfter(new Date(event.startsAt), new Date()))
-        .sort((a, b) => (a.startsAt < b.startsAt ? -1 : 1))
+        .sort(sorts.ascending)
     } else {
       return data
         .filter((event) => isBefore(new Date(event.startsAt), new Date()))
-        .sort((a, b) => (a.startsAt > b.startsAt ? -1 : 1))
+        .sort(sorts.descending)
```

Comment:
Isn't this more declarative than the inline arrow function example from before? We can almost read this in a human language way: "sort by the descending sort".